bundles/org.openhab.binding.insteon/src/main/resources/OH-INF/thing/channels.xml

```diff
@@ -0,0 +1,1102 @@
+		<description>Light Level</description>
+		<state readOnly="true"/>
+	</channel-type>
+
+	<channel-type id="load">
+		<item-type>Switch</item-type>
+		<label>Load</label>
+		<description>Load State</description>
+		<state readOnly="true"/>
+	</channel-type>
+
+	<channel-type id="loadSense" advanced="true">
+		<item-type>Switch</item-type>
+		<label>Load Sense</label>
+		<description>Load Sense</description>
+	</channel-type>
+
+	<channel-type id="loadSenseBottom" advanced="true">
+		<item-type>Switch</item-type>
+		<label>Load Sense Bottom</label>
+		<description>Load Sense Bottom</description>
+	</channel-type>
+
+	<channel-type id="loadSenseTop" advanced="true">
+		<item-type>Switch</item-type>
+		<label>Load Sense Top</label>
+		<description>Load Sense Top</description>
+	</channel-type>
+
+	<channel-type id="lock">
+		<item-type>Switch</item-type>
+		<label>Lock</label>
+		<description>Lock</description>
+		<category>Lock</category>
+	</channel-type>
+
+	<channel-type id="lowBattery">
+		<item-type>Switch</item-type>
+		<label>Low Battery</label>
+		<description>Low Battery Alert</description>
+		<category>LowBattery</category>
+		<state readOnly="true"/>
+	</channel-type>
+
+	<channel-type id="momentaryDuration" advanced="true">
+		<item-type>Number:Time</item-type>
+		<label>Momentary Duration</label>
+		<description>Momentary Duration</description>
+		<state min="0.2" max="25" step="0.1" pattern="%.1f %unit%"/>
+	</channel-type>
+
+	<channel-type id="monitorMode" advanced="true">
+		<item-type>Switch</item-type>
+		<label>Monitor Mode</label>
+		<description>Monitor Mode</description>
+	</channel-type>
+
+	<channel-type id="motion">
+		<item-type>Switch</item-type>
+		<label>Motion</label>
+		<description>Motion Detected</description>
+		<category>Motion</category>
+		<state readOnly="true"/>
+	</channel-type>
+
+	<channel-type id="onLevel" advanced="true">
+		<item-type>Dimmer</item-type>
+		<label>On Level</label>
+		<description>On Level</description>
+	</channel-type>
+
+	<channel-type id="operationMode" advanced="true">
+		<item-type>String</item-type>
+		<label>Operation Mode</label>
+		<description>Switch Operation Mode</description>
+		<state>
+			<options>
+				<option value="LATCHING">Latching</option>
+				<option value="SINGLE_MOMENTARY">Single Momentary</option>
+				<option value="DUAL_MOMENTARY">Dual Momentary</option>
+			</options>
+		</state>
+	</channel-type>
+
+	<channel-type id="outletBottom">
+		<item-type>Switch</item-type>
+		<label>Outlet Bottom</label>
+		<description>Outlet Bottom</description>
+		<category>PowerOutlet</category>
+	</channel-type>
+
+	<channel-type id="outletTop">
+		<item-type>Switch</item-type>
+		<label>Outlet Top</label>
+		<description>Outlet Top</description>
+		<category>PowerOutlet</category>
+	</channel-type>
+
+	<channel-type id="powerUsage">
+		<item-type>Number:Power</item-type>
+		<label>Power Usage</label>
+		<description>Power Usage in Watts</description>
```

Comment:
Please remove unit in description as it depends on the configured unit

Reply:
Done.